### PR TITLE
OOT: Drain call is now invoked with the effective timeout

### PR DESCRIPTION
**Pull request by prashanth26**
Drain call was taking in the default drain timeout of 12hours. This change reverts the value to the effective value.

**What this PR does / why we need it**:
The drain call was made with the default timeout of 12hours. This PR reverts this change to use the effective timeout value to 2hours. This also makes the default drain timeout to 2hours.  

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```noteworthy user
Reduces default drain timeout from 12hours to 2hours.   
```
```improvement operator
OOT: Drain call was taking in the default drain timeout of 12hours. This change reverts the value to the effective value.
```

### CODE REVIEW

File: pkg/util/provider/machinecontroller/machine_util.go [hunk 1]
```diff
@@ -911,7 +911,7 @@ func (c *controller) drainNode(deleteMachineRequest *driver.DeleteMachineRequest
 
 			drainOptions := drain.NewDrainOptions(
 				c.targetCoreClient,
-				timeOutDuration,
+				timeOut,
```
Comment (hardikdr) [inline pkg/util/provider/machinecontroller/machine_util.go:914]: Is it that MCM restarts can reset the drainTimeout? I hope there are no side-effects from it.
Does this also need to be fixed in-tree then? https://github.com/gardener/machine-controller-manager/blob/master/pkg/controller/machine.go#L708 ?

Comment (prashanth26) [reply pkg/util/provider/machinecontroller/machine_util.go:914]: Oops. Nice catch. Wait if it's working as expected with in-tree why the difference in OOT. I guess the issue is something else then. Let me re-check.

**Comment by prashanth26** [comment]
It seems like something else is causing the issue. Let me check it out. I will try to write a test to reproduce it. 

My bad above was the wrong RCA.

**Comment by prashanth26** [comment]
Closing in favor of https://github.com/gardener/machine-controller-manager-provider-gcp/issues/9.

**Pull request closed without merging**